src/instructlab/model/accelerated_train.py

```diff
@@ -449,7 +490,14 @@ def _training_phase(
         f"TrainingArgs for current phase: {pprint.pformat(train_args)}", fg="cyan"
     )
 
-    run_training(train_args=train_args, torch_args=torch_args)
+    try:
```

Comment:
instead of _run_phase including bookkeeping, we could remove the bookkeeping and reuse that function here.

Reply:
I see your point here but I'd rather not do this. The point of `_run_phase` was to simplify the flow within the multi-phase training. @danmcp maybe you had some thoughts here?

Reply:
I am probably missing something, but looking at the current version closer, I am not sure why we need the try/except logic here at all.  Isn't this only called by _run_phase which also has the RuntimeError handling added?  There is a separate call to `run_training` in `accelerated_train` which does seem like it needs the exception handling instead.

Reply:
@danmcp Good point, I'll remove this line altogether.

Reply:
@JamesKunstle @danmcp I've removed this try/catch and moved it to the other `run_training` invocation per your suggestions. Please take a look and let me know if you spot any problems.